beta-src/src/components/ui/WDInfoPanel.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import WDPopover from "./WDPopover";
+import WDCountryTable from "./WDCountryTable";
+import WDVoteButtons from "./WDVoteButtons";
+import { CountryTableData } from "../../interfaces/CountryTableData";
+import { UserData } from "../../interfaces/UserData";
+import getDevice from "../../utils/getDevice";
+
+interface WDInfoPanelProps {
+  countries: CountryTableData[];
+  userData: UserData;
+}
+
+const WDInfoPanel: React.FC<WDInfoPanelProps> = function ({
+  countries,
+  userData,
+}): React.ReactElement {
+  const [voteState, setVoteState] = React.useState(userData.votes);
+  const [isOpen, setIsOpen] = React.useState(false);
+  const device = getDevice({ width: 1300, height: 900 });
```

Comment:
Bryan defined the viewport in WDMain component where the viewport is changed every time the window is resized. So the device should still be a prop passing to this component and the CountryTable component's padding will change when different device is defined.

Reply:
Added there for testing. Forgot to remove :D